subsys/bluetooth/host/conn.c

```diff
@@ -524,6 +562,7 @@ static bool send_frag(struct bt_conn *conn, struct net_buf *buf, uint8_t flags,
 		/* Roll back the pending TX info */
 		if (tx) {
 			sys_slist_find_and_remove(&conn->tx_pending, &tx->node);
+			tx_data(buf)->tx = tx;
```

Comment:
I was confused when reading the comment above. Then I realized what the comment was referring to when I read this line later. I think I would have had an easier time reading the code if the part about the reset was in comment here.